crates/providers/providers-alloy/src/beacon_client.rs

```diff
@@ -88,6 +87,26 @@ pub trait BeaconClient {
     ) -> Result<Vec<BoxedBlobWithIndex>, Self::Error>;
 }
 
+/// blob_versioned_hash computes the versioned hash of a blob.
```

Comment:
We can leverage the rust doc comments to point to the method name, so that when we change the name we need to change the associated doc.
```suggestion
/// [`blob_versioned_hash`] computes the versioned hash of a blob.
```